apps/transport/priv/gettext/en/LC_MESSAGES/validations.po

```diff
@@ -41,7 +41,7 @@ msgstr "This is the automated evalation of the GTFS datafile. "
 "This report can be shared with"
 
 #, elixir-format
-msgid "this permanent link"
+msgid "permanent link to this validation"
```

Comment:
J'avais changé pour que la phrase soit "This report can be shared with this permanent link".
Ici ça devient "This report can be shared with permanent link to this validation"

![image](https://user-images.githubusercontent.com/15341118/74346833-71f9ca80-4db0-11ea-866c-f9a2ab7b2f4c.png)

Reply:
hum j'ai peut etre (surement :roll_eyes: ) merdé au rebase, mais sur master je ne vois `"permanent link to this validation"` et pas `"this permanent link"` :thinking:

Reply:
ah je crois que j'ai pigé :tired_face: